docs/usage/gitlab-bot-security.md

```diff
@@ -34,7 +34,7 @@ The following research notes may help you to assess the GitLab bot security risk
 ### Public projects only
 
 If a bot service is run on public projects only, then the risk of private project data being accessed by unauthorized users is zero.
-However, malicious users can still spoof or spam packages to any other public project they themselves are not a member of, so that rules out this approach for a public hosted service.
+But malicious users can still spoof or spam packages to any other public project they themselves are not a member of, so that rules out this approach for a public hosted service.
```

Comment:
```suggestion
But malicious users can still spoof or spam packages to any other public project they are not a member of, so that rules out this approach for a public hosted service.
```

We can get rid of `themselves`.